lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -37,24 +38,32 @@ class _FindClosestFacilityFromPointSampleState
       'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0');
   static final _incidentsLayerUri = Uri.parse(
       'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0');
+  // Create a task for the closest facility service.
   final _closestFacilityTask = ClosestFacilityTask.withUrl(Uri.parse(
       'https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility'));
-
+  // Create a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
+  // Create a graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
-  bool _isRouteSolved = false;
-  bool _isInitialized = false;
+  // Create a flag to track whether the route has been solved.
+  bool _routeSolved = false;
+  bool _initialized = false;
+  // Create parameters for the closest facility task.
   late final ClosestFacilityParameters _closestFacilityParameters;
+  // Create a symbol for the route line.
   final _routeLineSymbol = SimpleLineSymbol(
       style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
+        top: false,
+        // Create a column with the map view and buttons to solve the routes and reset the graphics.
         child: Column(
           children: [
             Expanded(
+              // Create the map view.
```

Comment:
```suggestion
              // Add a map view to the widget tree and set a controller.
```
Use the standard comment from the skeleton here